### DISCUSSION TITLE
System.IO.Compression.Brotli not supported on Blazor webassembly

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/aspnetcore#49276] [Q&A] [answered]
I heard System.IO.Compression.Brotli isn't supported on the browser platform, is it really

**Comment by gfoidl** [comment, marked as answer]
According to [apisof.net](https://apisof.net/catalog/b17be566-b03e-8549-559d-cf3a237e5a85): yes (see under _OS Platforms_)
